tests/outputs/a_manusrev/refereeReports.yaml

```diff
@@ -0,0 +1,42 @@
+1902cl_actaArev:
+  claimed_found_what:
```

Comment:
these shouldn't be emtpy, but don't put fake values, so just empty strings and empty lists, etc., whatever the schema reuqires.